pkg/antctl/antctl.go

```diff
@@ -568,6 +569,22 @@ $ antctl get podmulticaststats pod -n namespace`,
 			},
 			transformedResponse: reflect.TypeOf(serviceexternalip.Response{}),
 		},
+		{
+			use:     "memberlist",
+			aliases: []string{"ml"},
+			short:   "Print state of memberlist cluster",
+			long:    "Print state of memberlist cluster of Antrea agent",
+			example: `  Get a memberlist
```

Comment:
Since there is only one usage, this example looks verbose, could be deleted

Reply:
deleted.